app/code/Magento/CatalogGraphQl/Model/Category/DepthCalculator.php

```diff
@@ -17,22 +20,54 @@ class DepthCalculator
     /**
      * Calculate the total depth of a category tree inside a GraphQL request
      *
+     * @param ResolveInfo $resolveInfo
      * @param FieldNode $fieldNode
      * @return int
      */
-    public function calculate(FieldNode $fieldNode) : int
+    public function calculate(ResolveInfo $resolveInfo, FieldNode $fieldNode) : int
```

Comment:
Can we swap the arguments in this method signature just so it is similar to other related public methods

Reply:
Do you mean the $resolveInfo in second place here?

Reply:
Yes, in other methods where you've added resolveInfo to the input args, it had been placed last, so I think it would be good to do the same here for consistency

Reply:
I will leave it for now as we decided in another comment.